<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeViewWithProgressBar" name="treeViewLeft">
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeViewWithProgressBar" name="treeViewRight">
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="moveButton">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyButton">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insertButton">
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="folderButton">
        <property name="text">
         <string>New folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewFolder"/>
    <addaction name="actionShowSize"/>
    <addaction name="actionShowType"/>
    <addaction name="actionShowDate"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewFolder">
   <property name="text">
    <string>New folder</string>
   </property>
  </action>
  <action name="actionShowSize">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show size</string>
   </property>
  </action>
  <action name="actionShowType">
   <property name="text">
    <string>Show type</string>
   </property>
  </action>
  <action name="actionShowDate">
   <property name="text">
    <string>Show date</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTreeViewWithProgressBar</class>
   <extends>QTreeView</extends>
   <header>qtreeviewwithprogressbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
